### ISSUE TITLE
More improvement of model accuracy

### ISSUE THREAD

**Issue by pratikraj244** [FaizanZaheerGit/StudentPerformancePrediction-ML#9]
data1 = pd.read_csv("/content/AI-Data.csv")
from sklearn.preprocessing import LabelEncoder
le = LabelEncoder()
for i in data2.columns:
  if data2[i].dtype == "object" or data2[i].dtype == "bool":
    data2[i] = le.fit_transform(data2[i])
x = data2.drop("Class", axis=1)
y = data2["Class"]
from sklearn.model_selection import train_test_split
x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=0.2, random_state=0)

# RANDOM FOREST CLASSIFIER
from sklearn.ensemble import RandomForestClassifier
from sklearn.metrics import accuracy_score,classification_report
model = RandomForestClassifier()
model.fit(x_train, y_train)
y_pred = model.predict(x_test)
print(accuracy_score(y_test,y_pred))
print(classification_report(y_test,y_pred)) # Getting Accuracy of 79%

0.7916666666666666
              precision    recall  f1-score   support

           0       0.71      0.71      0.71        28
           1       0.85      1.00      0.92        23
           2       0.80      0.73      0.77        45

    accuracy                           0.79        96
   macro avg       0.79      0.82      0.80        96
weighted avg       0.79      0.79      0.79        96

# CATBOOST CLASSIFIER
!pip install catboost
from catboost import CatBoostClassifier
model_CBR = CatBoostClassifier()
from sklearn.model_selection import cross_val_score, KFold
model_CBR.fit(x_train, y_train)
y_pred_cbr = model_CBR.predict(x_test)
print(accuracy_score(y_test,y_pred_cbr))
print(classification_report(y_test,y_pred_cbr))  # Accuracy of 83%

0.8333333333333334
              precision    recall  f1-score   support

           0       0.80      0.71      0.75        28
           1       0.88      1.00      0.94        23
           2       0.82      0.82      0.82        45

    accuracy                           0.83        96
   macro avg       0.84      0.85      0.84        96
weighted avg       0.83      0.83      0.83        96

Although in my opinion, the accuracy may increase more. 
THATS IT FROM MY SIDE, REST EVERYTHING IS FINE
